app/lua53/lobject.h

```diff
@@ -80,7 +80,7 @@
 ** wo is the offset of aligned word in bytes 0,4,8,..
 ** bo is the field within the word in bits 0..31
 */
-#ifdef LUA_USE_ESP
+#if defined(LUA_USE_ESP8266)
```

Comment:
Another one that also applies to the ESP32?

Reply:
As previous comment. Perhaps we need to do that review sooner rather than later.